### PR TITLE
Consultant edit page and other small improvements

### PR DESCRIPTION
**Pull request by maurelio1234**
- Consultant edit page
- Combobox for statuses, other fields non editable (does not make sense)
- Make save work
- Make add page (not sure it works yet)
- Remove banner component... it won't be used anyway

image

image
vivExpertise

### CODE REVIEW

File: viv-wallet-app/src/views/ConsultantEdit.vue [hunk 1]
```diff
@@ -0,0 +1,143 @@
+<template>
+    <div class="consultantEdit">
+        <section v-if="errored">
+            <p>We're sorry, we're not able to retrieve this information at the moment, please try back later</p>
+        </section>
+        <section v-else>
+            <h2>Editer consultant</h2>
+            <div v-if="loading">Loading...</div>
```
Comment (nicolaserny) [inline viv-wallet-app/src/views/ConsultantEdit.vue:8]: Use our loading component instead of recoding the loading/error behavior

**Review by nicolaserny — CHANGES_REQUESTED** [review]
+ build error

File: viv-wallet-app/src/views/ConsultantEdit.vue [hunk 2]
```diff
@@ -0,0 +1,195 @@
...
+                this.consultant = await this.consultantsService.getConsultant(this.consultantId);
+            } else {
+                this.loading = true;
+                const consultantsInExpertise = await this.consultantsService.getConsultants();
+                const allUsers = await this.usersService.getUsers();
+
+                this.usersNotAlreadyInExpertise = allUsers.filter(
+                    u => !consultantsInExpertise.find(u1 => u1.id === u.id)
+                );
+                this.consultant.startDate = new Date().toLocaleDateString("en-CA"); // the format should be YYYY-MM-DD
```
Comment (nicolaserny) [inline viv-wallet-app/src/views/ConsultantEdit.vue:125]: Not a huge fan of this hack. It's a sign that we should use a real date library (date-fns is light and efficient library). But we can live with it for now.

**Review by nicolaserny — APPROVED** [review]

File: viv-wallet-app/src/views/ConsultantEdit.vue (hunk 2 above)
Comment (maurelio1234) [reply viv-wallet-app/src/views/ConsultantEdit.vue:125]: me neither... but I don't know which date format they are going o send us, I need to test it with the real backend.
but I still prefer a one line hack to a full library / dependency :)

**Pull request merged**